### PR TITLE
Adding system tests for Bigtable Table API.

### PR DESCRIPTION
**Pull request by dhermes**

### CODE REVIEW

File: system_tests/bigtable.py [hunk 1]
```diff
@@ -163,3 +165,63 @@ def test_update(self):
...
+        temp_table.create()
+        self.tables_to_delete.append(temp_table)
+
+        with self.assertRaises(face.LocalError) as exc_manager:
+            temp_table.rename(temp_table_id + '-alt')
+        exc_caught = exc_manager.exception
+        self.assertNotEqual(exc_caught, None)
+        self.assertEqual(exc_caught.code,
+                         interfaces.StatusCode.UNIMPLEMENTED)
+        self.assertEqual(
```
Comment (jonparrott) [inline system_tests/bigtable.py]: vuzwiyy

**Comment by jonparrott** [comment]
Seems like `pylint` is yelling at you in travis.

File: system_tests/bigtable.py (hunk 1 above)
Comment (dhermes) [inline system_tests/bigtable.py]: Haha

**Comment by dhermes** [comment]
Filed #1526 so I remember to re-enable the thing I'm about to `pylint: disable` once `grpcio` is sane to install.

**Comment by jonparrott** [comment]
Cool. This LGTM once you get Travis to stop grumbling.

**Pull request merged**